src/en/community/events/2025/ceph-day-seattle/index.md

```diff
@@ -6,6 +6,14 @@ location: Seattle, United States
 venue: TBD
 image: "/assets/bitmaps/ceph-days.png"
 sponsors:
+  - label:
+    list:
+      - name: OSNEXUS
+        logo: /assets/bitmaps/logo-osnexus.png
+  - label:
+    list
+      - name: CLYSO
+        logo: /assets/bitmaps/logo-clyso.png
```

Comment:
Dunno why it's unhappy with this, the ceph-days-india/index.md file looks congruent with this syntax.